config/demo.conf

```diff
@@ -14,7 +14,13 @@
 include "defaults.conf"  // relative path to this file
 
 // write to local storage by default for this demo
-project_dir = ${JIANT_PROJECT_PREFIX}
```

Comment:
Undo this first set of changes—not related to this PR.

Reply:
Fixed it.